src/main/java/net/minecraftforge/fmp/microblock/IMicroblockPlacementGrid.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.fmp.microblock;
+
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * Represents a grid for {@link IMicroblock} placement.
+ *
+ * @see IMicroblock
+ * @see MicroblockClass
+ */
+public interface IMicroblockPlacementGrid
```

Comment:
I'll just sideonly it, yes. That was temporary because I was considering adding some extra methods which didn't get added in the end. It's not called anywhere, it's just there to represent how a microblock class' placement grid should be rendered. The idea is that any mod will be able to come along and register their own microblock classes (like covers, fences or walls) and whichever mod adds the microblocks will automatically take care of that and register an item to handle it.